Updates h1 and h2 of Learning Hub (#175)
readthedocs/ethicalads.io: content/pages/learning-hub/publisher-guide.md

> title: Publisher's Guide to Hosting EthicalAds

Reviewer: ```suggestion
title: Publisher's Guide to Ad Placements for Developers
```

We don't need to rank for `EthicalAds`. This is a bit unwieldy but I think useful :)

Reviewer 2: ```suggestion
title: Publisher's Guide to Improving Developer Ads
```

slug: publisher-guide
description: A guide to help publishers get the most out of our network in terms of optimizing their ad placement and contextual targeting.

> subtitle: Good focus drives good ad revenue.

Reviewer 2: What does focus mean in this context? I know the previous version mentioned focus but then focus isn't mentioned in the body copy at all. I feel there is a bit of a mismatch here.

Reviewer 2: ```suggestion
subtitle: Optimizing your site's ads will improve your revenue.
```

> Our [EthicalAds client](https://ethical-ad-client.readthedocs.io/en/latest/) should get you started hosting ads, but for guidance on how to ensure your ads perform well, read on.

Reviewer 2: ```suggestion
Our [EthicalAds client](https://ethical-ad-client.readthedocs.io/en/latest/) gets you started with how to host ads. To ensure your site generates as much revenue from your ads as possible, you need to make sure your ad placement, targeting, and user experience compliment your content.
```

title: Publisher's Guide to Hosting EthicalAds
slug: publisher-guide
description: A guide to help publishers get the most out of our network in terms of optimizing their ad placement and contextual targeting.

> subtitle: Optimizing your site's ads will improve your revenue.

Reviewer 2: ```suggestion
subtitle: Maximize your site's ad revenue by complementing your user experience.
```
